core/node/house_keeper/src/prover/queue_reporter/fri_prover_queue_reporter.rs

```diff
@@ -39,45 +39,42 @@ impl PeriodicJob for FriProverQueueReporter {
         let mut conn = self.prover_connection_pool.connection().await.unwrap();
```

Comment:
Wait, isn't having prover connection contradicts with PR description? I'm kinda sure that this can be moved to prover job monitor as well.

Reply:
If the question is in metrics for autoscaler, it shouldn't be hard at all to change them in autoscaler config.